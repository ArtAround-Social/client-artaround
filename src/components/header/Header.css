

.section {
  overflow: hidden;
  background-color: rgba(16, 59, 202, 0.733);
  color: white;
}

.headerleft {
  float: left;
  padding: 5px;
}

.headerright {
  float: right
}