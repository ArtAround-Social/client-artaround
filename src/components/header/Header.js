import React from 'react';
import PropTypes from 'prop-types';
import styles from './Header.css';
import userImage from './user2.jpg';

//passing stupid travis

export default function Header({ user_type, name }) {
  return(
    <section className={styles.section}>
      <h3>{user_type} Account</h3>
      {name}
      <img src={userImage} alt='user png' height='40'/>
    </section>
  );
}

Header.propTypes = {
  user_type: PropTypes.string,
  name: PropTypes.string
};
