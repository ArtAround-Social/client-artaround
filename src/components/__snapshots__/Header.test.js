import React from 'react';
import { shallow } from 'enzyme';
import Header from '../header/Header';

describe('Header component', () => {
  it('renders header', () => {
    const wrapper = shallow(
      <Header
        user_type = 'Artist'
        handleLogout = {() => { }}
        name = 'vasily'
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
