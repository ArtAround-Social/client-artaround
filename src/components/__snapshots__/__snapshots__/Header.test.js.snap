// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders header 1`] = `
<section
  className="section"
>
  <h3>
    Artist
     Account
  </h3>
  vasily
  <img
    alt="user png"
    height="40"
    src="/path/image"
  />
</section>
`;
